src/js/dataMapping/navigation/menuOptions.jsx

```diff
@@ -130,5 +130,16 @@ export const downloadOptions = [
         newTab: false,
         enabled: true,
         externalLink: false
+    },
+    {
+        label: 'Metadata Download',
```

Comment:
One of the mockups linked from the ticket has "Dataset Metadata", another "Metadata Download" - probably a question for UX